and-cli-workspace.js

```diff
@@ -104,16 +104,14 @@ require("./command-runner").run(async () => {
 
     // Clone user forks of AndcultureCode repositories
     // -----------------------------------------------
-    if (program.usernames == null) {
-        shell.exit(0);
-    }
-
-    const usernames = program.usernames.split(",");
+    if (program.usernames != null) {
```

Comment:
Minor - we should have `StringUtils` available from `andculturecode-javascript-core` to value check this

Reply:
ah, good point